mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -28,19 +28,19 @@ pub enum CumulocityJsonError {
 }
 
 /// Converts from thin-edge Json to c8y_json
-pub fn from_thin_edge_json(input: &[u8]) -> Result<Vec<u8>, CumulocityJsonError> {
+pub fn from_thin_edge_json(input: &str) -> Result<String, CumulocityJsonError> {
     let timestamp = WallClock.now();
     let c8y_vec = from_thin_edge_json_with_timestamp(input, timestamp)?;
     Ok(c8y_vec)
 }
 
 fn from_thin_edge_json_with_timestamp(
-    input: &[u8],
+    input: &str,
     default_timestamp: DateTime<FixedOffset>,
-) -> Result<Vec<u8>, CumulocityJsonError> {
+) -> Result<String, CumulocityJsonError> {
     let mut serializer = serializer::C8yJsonSerializer::new(default_timestamp)?;
     let () = parse_utf8(input, &mut serializer)?;
-    Ok(serializer.bytes()?)
+    Ok(serializer.into_string()?)
```

Comment:
As of now we require the ? because we are converting the vec<u8> into the string. Agree, this can be done once we change the serializer code, maybe in another PR?